python/private/stage2_bootstrap_template.py

```diff
@@ -365,6 +369,21 @@ def main():
     print_verbose("initial environ:", mapping=os.environ)
     print_verbose("initial sys.path:", values=sys.path)
 
+    site_packages = os.path.join(sys.prefix, VENV_SITE_PACKAGES)
```

Comment:
nit: should we use `pathlib` for all path manipulations?

Reply:
Normally I would yes, but in this case, both values are strings and never go through additional path manipulations or other processing